themes/src/main/resources/theme/base/admin/resources/js/services.js

```diff
@@ -1398,7 +1398,7 @@ module.factory('TimeUnit2', function() {
 
     t.asUnit = function(time) {
         var unit = 'Minutes';
-        if (time) {
+        if (time != -1) {
```

Comment:
`if (time && time != -1)`

Reply:
It works the way it is today, but it's not obvious. What you propose will not work (-1 is a specific value, not what's used when it's undefined). Re-structuring it to make it more obvious.